adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -572,7 +572,7 @@ def publish(self, topic, msg, retain=False, qos=0):
             msg = str(msg).encode("ascii")
         elif isinstance(msg, str):
             msg = str(msg).encode("utf-8")
-        else:
```

Comment:
@PhearZero  Could you check `elif isinstance(msg, bytes)` and keep the `else:` after?

Reply:
@brentru Good to go, learned a new thing today ```pass``` in Python. Hopefully that cleans it up.

Reply:
perfect.